elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -48,7 +48,6 @@ async def verify_attached_bugs_cli(runtime: Runtime, verify_bug_status: bool, ad
 async def verify_attached_bugs(runtime: Runtime, verify_bug_status: bool, advisory_id_map: Dict[str, int], verify_flaws:
                                bool, no_verify_blocking_bugs: bool):
     validator = BugValidator(runtime, output="text")
-    await validator.errata_api.login()
```

Comment:
what login method does is add gssapi.SecurityContext and Authorization header, just need to make sure they are set before actually making the request, set them earlier doesn't help much